gpu-operator/install-gpu-operator-vgpu.rst

```diff
@@ -217,22 +220,21 @@ Configure the Cluster with the vGPU License Information and the Driver Container
       # 4 => for NVIDIA Virtual Compute Server
       FeatureType=1
 
-#. Rename the client configuration token file that you downloaded to ``client_configuration_token.tok``
-   using a command like the following example:
+#. Rename the client configuration token file that you downloaded to ``client_configuration_token.tok`` using a command like the following example:
 
    .. code-block:: console
 
       $ cp ~/Downloads/client_configuration_token_03-28-2023-16-16-36.tok client_configuration_token.tok
 
-   The file must be named ``client_configuraton_token.tok``.
+   The file must be named ``client_configuration_token.tok``.
 
 #. Create the ``gpu-operator`` namespace:
 
    .. code-block:: console
 
       $ kubectl create namespace gpu-operator
 
-#. Create a config map that is named ``licensing-config`` using the ``gridd.conf`` and ``client_configuration_token.tok`` files:
+#. Create a ConfigMap that is named ``licensing-config`` using the ``gridd.conf`` and ``client_configuration_token.tok`` files:
```

Comment:
sugg: Plain English is clearer.  If you are open to suggestion, I found the OCP docs team to have the best approach: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
Thanks for the suggestion. Yeah, I'll use "config map".